python/Quickstart.MD

```diff
@@ -142,192 +119,201 @@ Body of event request:
         "isSubmittedToPartner": false,
         "directSupplierImpact": {
             "businessImpact": "Business Impact",
-            "businessPriority": "HIGH"
-            }
+            "businessPriority": "MEDIUM"
         }
+    }
 }
+```  
+Note that your `False` values have been converted to `false` according to the JSON specification.  Everything looks good, so we can convert our dictionary to a JSON string.  
```

Comment:
Note that your `False` values have been converted to `false` according to the JSON specification. Now, can convert the dictionary to a JSON string.

Reply:
Reworded